content/doc/book/pipeline/docker.adoc

```diff
@@ -40,6 +40,16 @@ Practically any tool which can be
 link:https://hub.docker.com[packaged in a Docker container].
 can be used with ease by making only minor edits to a `Jenkinsfile`.
 
+== Path setup for mac OS users
+By default in jenkins "/usr/local/bin" not accessible in $PATH when jenkins execute pipeline in declarative mode, Requires to add path node in file "/usr/local/Cellar/jenkins-lts/2.176.3/homebrew.mxcl.jenkins-lts.plist" like this:
+
+<key>EnvironmentVariables</key>
+<dict>
+<key>PATH</key>  <string>/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Applications/Docker.app/Contents/Resources/bin/:/Users/Kh0a/Library/Group\Containers/group.com.docker/Applications/Docker.app/Contents/Resources/bin</string>
+</dict>
+
+Now restart jenkins using "brew services restart jenkins-lts" 
+
```

Comment:
I don't have a macOS machine to check these instructions.  Others would need to check them and report that they work.  I am skeptical that they won't work for most users because they include specific details for the person that wrote the instructions.

We should not include a reference to 2.176.3, an LTS version that is unsupported and is not installed on most user machines.  I don't know what the correct reference should be, but I am confident that it should not be 2.176.3

```suggestion
== Path setup for mac OS users

The `/usr/local/bin` directory is not included in the macOS `PATH` for Docker images by default.
If executables from `/usr/local/bin` need to be called from within Jenkins, then the `PATH` needs to be extended to include `/usr/local/bin`.
Requires to add path node in file "/usr/local/Cellar/jenkins-lts/2.176.3/homebrew.mxcl.jenkins-lts.plist" like this:

.Contents of homebrew.mxcl.jenkins-lts.plist
[source,xml]
----
<key>EnvironmentVariables</key>
<dict>
<key>PATH</key>  <string>/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Applications/Docker.app/Contents/Resources/bin/:/Users/XXX/Library/Group/Containers/group.com.docker/Applications/Docker.app/Contents/Resources/bin</string>
</dict>
----

Now restart jenkins using "brew services restart jenkins-lts" 

```

Reply:
@krisstern is a macOS user and may be able to assist with this review

Reply:
I think it makes sense, since macOS users can use `homebrew` to install `jenkins-lts` while installing Docker Desktop from [here](https://www.docker.com/products/docker-desktop/) which uses the default paths specified, except I spotted a typo in `/Users/XXX/Library/Group/Containers/group.com.docker/Applications/Docker.app/Contents/Resources/bin` which should be `/Users/XXX/Library/Group\ Containers/group.com.docker/Applications/Docker.app/Contents/Resources/bin`.

Other than that everything else looks okay.

Reply:
Thanks @krisstern .  I've revised my proposed edit to include the embedded space.

The very long line will cause the entire web page to have a horizontal scroll bar on the page even on very wide monitors.  I've reduced the line length slightly by making it two lines, but even that won't really solve the issue.

More investigation will be needed before we merge this or the page layout will be very poor after the merge

Reply:
Thanks @MarkEWaite @krisstern for verification of path setup for mac OS users, it looks great in UI of jenkins docs.